scripts/chronicle_dump/chronicle_dump.erl

```diff
@@ -150,6 +150,51 @@ sanitize_opt(Value) ->
             {error, invalid_function}
     end.
 
+binarify_output_item(Binary) when is_binary(Binary) -> Binary;
+binarify_output_item(Atom) when is_atom(Atom) -> atom_to_binary(Atom, latin1);
+binarify_output_item(Int) when is_integer(Int) -> integer_to_list(Int);
```

Comment:
Bit confusing - Int and List below aren't being converted to a binary. Is that intended?

Reply:
The change to support chronicle guts is patterned after the existing server dump-guts script. The manipulation of the data is to accommodate transfer from this erlang script to cbcollect_info python code and is copied from https://src.couchbase.org/source/xref/trinity/ns_server/scripts/dump-guts?r=e2878978#632-635
Notice that some items require special manipulations to fit a format agreeable between guts and cbcollect. An example is cluster_compat_version:

{cluster_compat_version,
                  iolist_to_binary(io_lib:format("~p", [CompatVer]))},

Reply:
might be able to just rename this function to exclude the term "binarify" (and use a different name...) which I think causes the confusion

Reply:
Done.